tests/regression/nextcloud-rule-exclusions-plugin/9508400.yaml

```diff
@@ -52,7 +52,7 @@ tests:
             uri: /login
             data: "redirect_url=/login/v2/grant?user=Esad%20Cetiner&stateToken=sample-token"
           output:
-            no_log_contains: |
+            no_log_contains: |-
               id "920230"|id "932(?:19|20)0"|id "94243[12]"
```

Comment:
Why we need doing this in tests? It's unreadable, cannot be searched for rule IDs and so on. Let's simply specify whole rule IDs without using regexes.